lineman/app/components/group_page/join_group_button/join_group_button.coffee

```diff
@@ -0,0 +1,23 @@
+angular.module('loomioApp').directive 'joinGroupButton', ->
+  scope: {group: '='}
+  restrict: 'E'
+  templateUrl: 'generated/components/group_page/join_group_button/join_group_button.html'
+  replace: true
+  controller: ($scope, AbilityService, ModalService, CurrentUser, Records, FlashService, MembershipRequestForm) ->
+    Records.membershipRequests.fetchMyPendingByGroup($scope.group.key)
+
+    $scope.isMember = ->
+      CurrentUser.membershipFor($scope.group)?
```

Comment:
Do we need isMember() and canJoinGroup()? Seems like canJoinGroup could cover both.